clippy_lints/src/swap.rs

```diff
@@ -172,15 +172,16 @@ fn check_manual_swap(cx: &LateContext<'_>, block: &Block<'_>) {
     }
 }
 
+#[allow(clippy::too_many_lines)]
 /// Implementation of the `ALMOST_SWAPPED` lint.
 fn check_suspicious_swap(cx: &LateContext<'_>, block: &Block<'_>) {
-    for w in block.stmts.windows(2) {
+    for stmt in block.stmts.windows(2) {
```

Comment:
While we're here we could change this to use

```suggestion
    for [first, second] in block.stmts.array_windows() {
```

Reply:
@Alexendoo Thank you so much for your review. I will fix this.